mappings/net/minecraft/entity/player/Player.mapping

```diff
@@ -5,12 +5,29 @@ CLASS gs net/minecraft/entity/player/Player
 	FIELD d playerContainer Ldw;
 	FIELD e container Ldw;
 	FIELD g score I
+	FIELD j handSwinging Z
+	FIELD k handSwingTicks I
 	FIELD m dimensionId I
 	FIELD u sleeping Z
+	FIELD v bedPosition Lbr;
+	METHOD D dropSelectedItem ()V
+	METHOD E updateEyeHeight ()V
 	METHOD G getHeldItem ()Liz;
+	METHOD J swingHand ()V
+	METHOD O hasSleptEnough ()Z
```

Comment:
I wonder why this is a method in beta 1.7.3... it isn't used for anything is it?

Reply:
sleeping?